apiary.apib

```diff
@@ -688,6 +688,13 @@ Get a list of departments.
                 + name: `Human Resources` (string)
                 + vat_number: `BE0899623035` (string)
                 + currency: `EUR` (string)
+                + emails (array)
+                    + (object)
+                        + type: `primary` (enum)
```

Comment:
this description looks weird, is it `type: 'string'` maybe?

Reply:
Within the the quotes is for example data, the type is at the back as `(enum)`